src/main/java/frc/robot/Robot.java

```diff
@@ -25,9 +31,18 @@ public class Robot extends TimedRobot {
    */
   @Override
   public void robotInit() {
+
+    try {
+      localizer = new Localizer();
+    } catch (IOException e) {
+      // TODO Auto-generated catch block
```

Comment:
Rethrow the error after this init so that if localizer fails the robot won't start up.